docs/usage/config-presets.md

```diff
@@ -232,6 +232,22 @@ Parameters are supported similar to other methods:
 }
 ```
 
+## Templating presets
+
+In order to provide flexible preset configurations, Renovate provides support using [templates](./templates.md) for preset names.
+
+### Example use-case
+
+The following example shows a self-hosted Renovate configuration for a company called "Renovators".
+The company is split into multiple organisations `foo`, `bar` and `baz`, where each organisation provides its opinionated shareable preset.
+Renovate is configured to load the preset of each organisation dynamically based on the repository path.
+
+```javascript
+module.exports = {
+  extends: ['local>{{ topLevelOrg }}/renovate'],
```

Comment:
are you sure this will work?

Reply:
I don't think it works currently as only `{}` is passed as the object